### PR TITLE
Widget Twitter account @zfmodules

### PR DESCRIPTION
**Pull request by gianarb**
Hi!
I opend a new twitter account [@zfmodules](https://twitter.com/zfmodules) for this site.
Ifttt tweet every new module updated

### CODE REVIEW

File: module/ZfModule/view/zf-module/helper/twitter-widget-view.phtml [hunk 1]
```diff
@@ -0,0 +1,3 @@
+<a class="twitter-timeline" href="https://twitter.com/zfmodules" data-widget-id="573451599546318849">Tweet di @zfmodules</a>
+<script>!function(d,s,id){var js,fjs=d.getElementsByTagName(s)[0],p=/^http:/.test(d.location)?'http':'https';if(!d.getElementById(id)){js=d.createElement(s);js.id=id;js.src=p+"://platform.twitter.com/widgets.js";fjs.parentNode.insertBefore(js,fjs);}}(document,"script","twitter-wjs");</script>
```
Comment (Ocramius) [inline module/ZfModule/view/zf-module/helper/twitter-widget-view.phtml]: Can you move this to a different file?

Comment (gianarb) [inline module/ZfModule/view/zf-module/helper/twitter-widget-view.phtml]: Not in a view helper?

Comment (Ocramius) [inline module/ZfModule/view/zf-module/helper/twitter-widget-view.phtml]: It's going to be reloaded every time at every page here. If you move it to `$this->headScript()` it's much better IMO

File: module/ZfModule/view/zf-module/helper/twitter-widget-view.phtml [hunk 2]
```diff
@@ -0,0 +1,2 @@
+<a class="twitter-timeline" href="https://twitter.com/zfmodules" data-widget-id="573451599546318849">Tweet di @zfmodules</a>
+<?php $this->headScript()->appendFile("/js/twitter-widget.js", "application/javascript");?>
```
Comment (gianarb) [inline module/ZfModule/view/zf-module/helper/twitter-widget-view.phtml]: @Ocramius in this way?

**Comment by snapshotpl** [comment]
Good idea. However we need to focus on tweet using api.

File: module/ZfModule/view/zf-module/helper/twitter-widget-view.phtml [hunk 3]
```diff
@@ -0,0 +1,2 @@
+<a class="twitter-timeline" href="https://twitter.com/zfmodules" data-widget-id="573451599546318849">Tweet di @zfmodules</a>
```
Comment (snapshotpl) [inline module/ZfModule/view/zf-module/helper/twitter-widget-view.phtml]: Keep account name in config

Comment (gianarb) [inline module/ZfModule/view/zf-module/helper/twitter-widget-view.phtml]: And widget id? :)

Comment (snapshotpl) [inline module/ZfModule/view/zf-module/helper/twitter-widget-view.phtml]: What is it? It's user id?

Comment (gianarb) [inline module/ZfModule/view/zf-module/helper/twitter-widget-view.phtml]: Every twitter's widget has a id :)

File: module/ZfModule/view/zf-module/helper/twitter-widget-view.phtml (hunk 2 above)
Comment (localheinz) [inline module/ZfModule/view/zf-module/helper/twitter-widget-view.phtml]: @gianarb 

Or with single quotes! :trollface:

File: module/ZfModule/view/zf-module/helper/twitter-widget-view.phtml (hunk 3 above)
Comment (localheinz) [inline module/ZfModule/view/zf-module/helper/twitter-widget-view.phtml]: @gianarb 

> Tweet di @zfmodules

Is this Italian?

File: module/ZfModule/view/zf-module/helper/twitter-widget-view.phtml (hunk 2 above)
Comment (gianarb) [inline module/ZfModule/view/zf-module/helper/twitter-widget-view.phtml]: grrrrrrr :imp: I change it sorry

File: module/ZfModule/view/zf-module/helper/twitter-widget-view.phtml (hunk 3 above)
Comment (gianarb) [inline module/ZfModule/view/zf-module/helper/twitter-widget-view.phtml]: Sorry I copied code generated by my italian Twitter :)

Comment (snapshotpl) [inline module/ZfModule/view/zf-module/helper/twitter-widget-view.phtml]: So add it also to config :)

Comment (gianarb) [inline module/ZfModule/view/zf-module/helper/twitter-widget-view.phtml]: In my opinion in configuration this params not works.. If I insert this params how arguments of invoke how do you think?

Comment (gianarb) [inline module/ZfModule/view/zf-module/helper/twitter-widget-view.phtml]: @Ocramius @ins0 @localheinz opinion about this stuff? :) Thanks

**Comment by gianarb** [comment]
Rebased.. feedback? In my opinion this repository is for zfmodules site not for an abstract zf modules repository.. :) Move this data into the config is unnecessary

**Comment by localheinz** [comment]
@gianarb 

I'm not so sure of the actual value of this as I think that you can already see the latest modules when you're on the site. Also, the right column seems a bit crowded with the widget! Since it pushes down the list of newly registered developers, it also decreases the value of that list.

What do you think? 

Maybe a simple [follow button](https://dev.twitter.com/web/follow-button) in the header is sufficient?

**Comment by gianarb** [comment]
Yes, this is a zfmodules account, in this moment publish only new modules
because is automatic :P But If you would send other tweets is not a
problem..
I have tried to move this widget after contributors box but maybe this
module have a layout's bug :) If the order change is a good proposal can
you try it?

2015-03-09 12:32 GMT+01:00 Andreas Möller <notifications@github.com>:

> @gianarb <https://github.com/gianarb>
>
> I'm not so sure of the actual value of this as I think that you can
> already see the latest modules when you're on the site. Also, the right
> column seems a bit crowded with the widget! Since it pushes down the list
> of newly registered developers, it also decreases the value of that list.
>
> What do you think?
>
> Maybe a simple follow button <https://dev.twitter.com/web/follow-button>
> in the header is sufficient?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/zendframework/modules.zendframework.com/pull/465#issuecomment-77838000>
> .
>



-- 
Gianluca Arbezzano
www.gianarb.it

**Comment by gianarb** [comment]
screen shot 2015-03-21 at 14 03 38

**Comment by gianarb** [comment]
Ping @localheinz @Ocramius 

:) :)

**Comment by gianarb** [comment]
ping @localheinz can you push this pr or there are a problems? :smile:

**Comment by localheinz** [comment]
Thank you, @gianarb!

Only problem here was a lack of time, going to deploy this later tonight when I'll sit together with @ins0, ok?

**Comment by gianarb** [comment]
Np :) thank you!! :)

2015-03-26 14:04 GMT+01:00 Andreas Möller <notifications@github.com>:

> Merged #465
> <https://github.com/zendframework/modules.zendframework.com/pull/465>.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/zendframework/modules.zendframework.com/pull/465#event-265822225>
> .
>



-- 
Gianluca Arbezzano
www.gianarb.it

**Pull request merged**